config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1845,7 +1845,7 @@ presubmits:
         - --gcp-zone=us-west1-b
         - --parallelism=1
         - --focus-regex=\[Feature:HugePages\]
-        - --skip-regex=
+        - --skip-regex=""
```

Comment:
Both has the same effect. I suppose you wanted to make it more obvious that the empty string gets set intentionally?

Reply:
yes, this is the intent. I can remove unnecessary changes if you or other reviewes think this is best.

Reply:
No strong preference from me.

I went even further and added a warning in https://github.com/kubernetes/test-infra/blob/245a3b241ee0aa29d628a9c9fbf4c3d35bfe409d/config/jobs/kubernetes/sig-node/dra.jinja#L93